src/kernels/linux-hardened.sh

```diff
@@ -64,9 +64,9 @@ update_linux_hardened_pkgbuilds() {
 update_linux_hardened_git_pkgbuilds() {
     pkg_list=("spl-linux-hardened-git" "zfs-linux-hardened-git")
     kernel_version=${kernel_version_git}
-    kernel_version_full=$(kernel_version_full ${kernel_version_git})
-    kernel_version_full_pkgver=$(kernel_version_full_no_hyphen ${kernel_version_git})
-    kernel_version_major=${kernel_version_git%-*}
+    kernel_version_full=$(kernel_version_full ${kernel_version})
+    kernel_version_full_pkgver=$(kernel_version_full_no_hyphen ${kernel_version})
+    kernel_version_major=${kernel_version%-*}
```

Comment:
I am uncertain how this change will affect the build scripts. But I do want to take a deeper look at this. For now I will hold off on merging these changes until #139 is ready to be worked on.